virttest/utils_test/__init__.py

```diff
@@ -1874,17 +1886,23 @@ def __init__(self, stress_type, params, download_url="", make_cmds="",
         self.timeout = 60
         self.stress_type = stress_type
         stress_cmds = stress_cmds or stress_type
-        self.stress_cmds = self.params.get('stress_cmds_%s' % stress_type, stress_cmds)
-        self.stress_args = self.params.get("%s_args" % stress_type, stress_args)
+        self.stress_cmds = self.params.get(
+            'stress_cmds_%s' % stress_type, stress_cmds)
+        self.stress_args = self.params.get(
+            "%s_args" % stress_type, stress_args)
         self.download_url = self.params.get('download_url_%s' % stress_type,
                                             download_url)
-        self.download_type = self.params.get('download_type_%s' % stress_type, download_type)
+        self.download_type = self.params.get(
+            'download_type_%s' % stress_type, download_type)
         self.base_name = self.download_url.split("/")[-1]
-        self.make_cmds = self.params.get('make_cmds_%s' % stress_type, make_cmds)
+        self.make_cmds = self.params.get(
+            'make_cmds_%s' % stress_type, make_cmds)
         self.make_cmds = self.make_cmds or './configure && make install'
-        self.uninstall_cmds = self.params.get('uninstall_cmds_%s' % stress_type, uninstall_cmds)
+        self.uninstall_cmds = self.params.get(
+            'uninstall_cmds_%s' % stress_type, uninstall_cmds)
         self.uninstall_cmds = self.uninstall_cmds or './configure && make uninstall'
-        self.work_path = self.params.get('%s_work_path' % stress_type, work_path)
+        self.work_path = self.params.get(
+            '%s_work_path' % stress_type, work_path)
```

Comment:
These changes are good, for better readability it can be something like this,
```python
self.work_path = self.params.get('%s_work_path' %
                                 stress_type, work_path)
```